src/cli/server/http_2.rs

```diff
@@ -53,12 +65,13 @@ pub async fn start_http_2(
             .or(Err(anyhow::anyhow!("Failed to send message")))?;
     }
 
-    let server: std::prelude::v1::Result<(), hyper::Error> =
-        if sc.blueprint.server.enable_batch_requests {
-            builder.serve(make_svc_batch_req).await
-        } else {
-            builder.serve(make_svc_single_req).await
-        };
+    let server: std::prelude::v1::Result<(), hyper::Error> = if sc.blueprint.server.enable_jit {
+        builder.serve(jit_req).await
```

Comment:
We should use the same request handler and not create a new one here.

Reply:
We can't do that because serialization for JIT request is entirely different